### PR TITLE
[CosmosDB] Bump up management SDK package from 8.0.0 to 9.0.0

### PR DESCRIPTION
**Pull request by kr-santosh**
**Related command**

**Description**

This PR bumps up management sdk package (azure-mgmt-cosmosdb) from 8.0.0 to 9.0.0.  This package now support client side encryption which was earlier part of preview package and now in stable release. Also fixes issue with CosmosDB container update for containers configured with client encryption policy.

**Testing Guide**

**History Notes**

[CosmosDB] `az cosmosdb container create`: GA Client Side Encryption feature
[CosmosDB] `az cosmosdb container update`: Fix updates failure for containers with client encryption policy

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CosmosDB

**Comment by evelyn-ys** [comment]
Can you add the history notes in the format of :

- `az cosmosdb xxx`: GA XXX feature
- `az cosmosdb yyy`: Fix the issue of ......

**Comment by kr-santosh** [comment]
> Can you add the history notes in the format of :
> 
> * `az cosmosdb xxx`: GA XXX feature
> * `az cosmosdb yyy`: Fix the issue of ......

Fixed

**Comment by evelyn-ys** [comment]
Hi @kr-santosh, I merged another PR bumping cosmosDB from 8.0.0 to 9.0.0 and it brings many conflicts to your PR..... Can you help resolve?

**Comment by kr-santosh** [comment]
> Hi @kr-santosh, I merged another PR bumping cosmosDB from 8.0.0 to 9.0.0 and it brings many conflicts to your PR..... Can you help resolve?

Fixed. Thanks

**Comment by kr-santosh** [comment]
Closed it by mistake

**Review by evelyn-ys — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py [hunk 1]
```diff
@@ -640,6 +640,9 @@ def cli_cosmosdb_sql_container_update(client,
     sql_container_resource.unique_key_policy = sql_container.resource.unique_key_policy
     sql_container_resource.conflict_resolution_policy = sql_container.resource.conflict_resolution_policy
 
+    """ client encryption policy is immutable"""
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py:643]:
```suggestion
    # client encryption policy is immutable
```

**Review by evelyn-ys — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**